Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1013,6 +1013,11 @@ LayoutUnit RenderFlexibleBox::crossAxisScrollbarExtent() const
     return isHorizontalFlow() ? horizontalScrollbarHeight() : verticalScrollbarWidth();
 }
 
+bool RenderFlexibleBox::flexItemIsScrollContainerCrossAxis(const RenderBox& child) const
+{
+    return isHorizontalFlow() ? child.isScrollContainerY() : child.isScrollContainerX();
+}
+
```

Comment:
Does this need a dedicated function?

Reply:
Not necessarily. Was mostly just for readability but after inlining the logic into the call site its still seems pretty readable